src/io/flutter/run/daemon/DaemonApi.java

```diff
@@ -404,6 +408,14 @@ RestartResult parseResult(JsonElement result) {
     }
   }
 
+  private static class DaemonShutdown extends Params<Boolean> {
```

Comment:
no need for an inner class. Nust write as a Lambda.

Reply:
I think I'm not doing this correctly - see above (ln 77). A class is required that implements parseResult, is there a way to do that with a lambda?

Reply:
ah you are right. I missed the extends.